docs/source/usage/tutorials/Bandits/index.rst

```diff
@@ -0,0 +1,11 @@
+Bandits
+============
+
+.. toctree::
+    :maxdepth: 1
+
+    Tutorial_on_bandits
+    Using_UCB
+    Using_Thompson_Sampling
+    Using_Bayesian_Method
+    Using_Gradient_Method
```

Comment:
Do we need these to be named `Using-*`?

Reply:
Even I thought it was redundant to use `Using-*` under tutorials, but the other files were named that way. I can change that and also the heading in those files, because that's what will show up as the text on the hyperlinks on the site, I think.

Reply:
Hmm, I guess it started with the A2C PR. We can do without the "Using" because it's in usage/tutorials anyway.

Reply:
@sampreet-arthi I've done it. I think everything is resolved now.